app/src/test/java/com/brewfind/BreweryDataUnitTest.kt

```diff
@@ -62,7 +61,7 @@ class BreweryDataUnitTest {
     }
 
     private fun whenISearchForGarbage() {
-        mvm.fetchBreweries("huahhudnwjnajkbqwe")
+        mvm.fetchBreweriesByName("huahhudnwjnajkbqwe")
```

Comment:
Fixing an error which would call a nonexistent string in another method. Changed it by having it call a new method with the purpose of having the string that was needed.